src/pages/billing/List/ProjectListItem.test.ts

```diff
@@ -0,0 +1,154 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { axe } from 'jest-axe'
+import { h } from 'react-hyperscript-helpers'
+import { WorkspaceWrapper } from 'src/libs/workspace-utils'
+import { ProjectListItem, ProjectListItemProps } from 'src/pages/billing/List/ProjectListItem'
+import { BillingProject } from 'src/pages/billing/models/BillingProject'
+
+
+type WorkspaceUtilsExports = typeof import('src/components/workspace-utils')
+jest.mock('src/components/workspace-utils', (): WorkspaceUtilsExports => {
+  return {
+    ...jest.requireActual('src/components/workspace-utils'),
+    useWorkspaces: jest.fn().mockReturnValue({
+      workspaces: [{
+        workspace: { namespace: 'aDifferentProject', name: 'testWorkspaces', workspaceId: '6771d2c8-cd58-47da-a54c-6cdafacc4175' },
+        accessLevel: 'WRITER'
+      }] as WorkspaceWrapper[],
+      refresh: () => Promise.resolve(),
+      loading: false,
+    })
+  }
+})
+
+// Mocking for using Nav.getLink
+jest.mock('src/libs/nav', () => ({
+  ...jest.requireActual('src/libs/nav'),
+  getPath: jest.fn(() => '/test/'),
+  getLink: jest.fn(() => '/')
+}))
+
+describe('ProjectListItem', () => {
+  let billingProject: BillingProject
+  let projectListItemProps: ProjectListItemProps
+  let listRoot
+
+  beforeEach(() => {
+    document.body.innerHTML = ''
+    listRoot = document.createElement('div')
+    listRoot.setAttribute('role', 'list')
```

Comment:
Need to be adding the list item to an element with the correct role for a11y tests to pass.